Java/src/main/java/net/hypixel/api/HypixelAPI.java

```diff
@@ -40,19 +42,33 @@
 
     private final ExecutorService executorService;
     private final HttpClient httpClient;
+    private final RateLimiter rateLimiter;
 
     public HypixelAPI(UUID apiKey) {
         this.apiKey = apiKey;
 
         this.executorService = Executors.newCachedThreadPool();
         this.httpClient = HttpClientBuilder.create().build();
+        this.rateLimiter = new RateLimiter(DEFAULT_MAX_REQUESTS_PER_MINUTE);
     }
 
     /**
-     * Shuts down the internal executor service
+     * Set how many requests this API instance is allowed to make in one minute.
+     * <p>If more requests attempt to pass past this limit in one minute,
```

Comment:
Is that <   p > supposed to be there?

Reply:
It is, it forces a gap between the last line and the next